gh-101979: fix a bug that parentheses in metavar argument of add_argument() were dropped (#102318)
python/cpython: Misc/NEWS.d/next/Library/2023-02-28-09-52-25.gh-issue-101979.or3hXV.rst

> Fix a bug that parentheses in metavar argument of :func:`add_argument()` were

Reviewer: ```suggestion
Fix a bug where parentheses in the ``metavar`` argument to :meth:`argparse.ArgumentParser.add_argument` were
```
